src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -800,15 +805,48 @@ class EntityApiServiceSpec extends ApiServiceSpec {
         assertResult(StatusCodes.OK) {
           status
         }
+
+        def attrToString(a: Attribute): String = {
```

Comment:
pulled the inline function in sortBy out to a real one because now it's recursive